packages/cashier/src/pages/payment-agent/payment-agent-deposit/payment-agent-deposit.tsx

```diff
@@ -1,14 +1,26 @@
-import { PropTypes as MobxPropTypes } from 'mobx-react';
-import { toJS } from 'mobx';
-import PropTypes from 'prop-types';
 import React from 'react';
+import { toJS } from 'mobx';
+import { RootStore, TReactChangeEvent } from 'Types';
 import { Accordion, DesktopWrapper, Dropdown, MobileWrapper, SelectNative, Text } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 import { isMobile } from '@deriv/shared';
 import { connect } from 'Stores/connect';
 import PaymentAgentDetails from '../payment-agent-details';
 
-const PaymentAgentDeposit = ({ onChangePaymentMethod, payment_agent_list, selected_bank, supported_banks }) => {
+type TPaymentAgentDepositProps = {
+    onChangePaymentMethod: (e: { target: { name?: string; value?: string | number } }) => void;
+    // TODO: use PaymentagentList['list'] if phone_numbers changed to phones
+    payment_agent_list: Array<{ name: string; email: string; phones: string; urls: string }>;
```

Comment:
@hamid-deriv I would suggest extracting these two into a separate type for better readability.

Something like:
```suggestion
    onChangePaymentMethod: TOnPaymentMethodChangeCallback;
    payment_agent_list: TPaymentAgant[];
```